### DISCUSSION TITLE
Migration from built-in integration not successful

### DISCUSSION THREAD

**Discussion by teddy-rpi** [ekutner/home-connect-hass#85] [Community Support] [answered]
Hi there, 
I discovered your integration and it is exactly what I was looking for, as the built-in integration misses some important features for my home automation. However I have no sucess in migrating from the built-in integration to yours. What I have tried so far: 
- read the instructions and the faq
- downloaded the integration via HACS
- disabled the built-in integration
- commented out settings of home_connect integration in configuration.yaml
- used old client_id and client_secret for home_connect_alt settings in configuration.yaml
- turned on dishwasher and washing machine
- restarted home assistant
No luck
- waited 24h and restarted home assistant
No luck
- registered new application under home connect development
- tried again with new client_id and new client_secret
Unfortunately nothing happened. No entities are created, no errors or warnings in the logs.
Do you have any clue for me what else to try to get this thing working?
Thanks a lot!
Teddy

**Comment by ekutner** [comment]
Did you manage to go through the login flow and get a success message in HA?
If not then make sure you follow the setup procedure. Note that there have been changes to the way OAuth works in May `22 (or something around that time) release of HA, so if you're using an older app you probably need to update its configuration. 
If you did get the success message at the end of the login flow and still not seeing any appliances then make sure your appliances are visible to the mobile app when the wifi on the phone is turned OFF.

**Comment by teddy-rpi** [comment, marked as answer]
Thanks for your help! I finally successfully installed the integration. What I didn't realize was, that after downloading via HACS and adding the credentials in configuration.yaml I also had to add your integration via the GUI (Settings-Integrations-Add Integration) as if it was a regular integration. All other Hacs Integrations that I previously installed worked without this step. Therefore the login flow you described was never initialized. Very happy!
